### ISSUE TITLE
Crash - application failed to initialize properly (0xc0000142)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/omaha#11]
```
This is reporting http://crbug.com/15509 upstream.

I frequently encounter this error on a Windows XP machine. An error dialog 
box pops up with title "GoogleUpdate.exe - ApplicationError" and message 
"The 
application failed to initialize properly (0xc0000142). Click on OK to 
terminate the application.".

It seems to appear periodically, with frequency of ~2 hours (very 
approximate).

The error disappeared after running Windows Update, so unfortunately I 
don't have a good reproduction case for now. But maybe you can extract 
something from this comment by dhw:

I found this article from Microsoft that mentions error 0xc0000142 on XP; I 
don't know 
if it applies.  The article mentions for console-based applications.  I 
wonder if 
GoogleUpdate.exe is considered a console-based application?  The article 
is:

   http://support.microsoft.com/kb/950312

In short, the article includes a hotfix for Winsrv.dll for XP SP2 and SP3 
systems "that 
are experiencing this specific problem".
```

Original issue reported on code.google.com by `phajdan.jr@chromium.org` on 16 Jul 2009 at 11:25
